src/sentry/static/sentry/app/components/bases/pluginComponentBase.jsx

```diff
@@ -74,6 +74,11 @@ class PluginComponentBase extends React.Component {
     if (this.state.state == FormState.SAVING) {
       return;
     }
+    // XXX(dcramer): due to the way all of these hooks are implemented, we cant
+    // easily apply semantics in things like onSubmit
+    if (args[0].preventDefault) {
```

Comment:
The form onsubmit hook wasn't working as the event fired and it posted as a normal form.